modules/ROOT/pages/message-logging-policy.adoc

```diff
@@ -85,8 +85,58 @@ Note: If the payload is used as part of the message log, do not configure the Li
 |After Calling API |Set to cause your logging policy to log messages after the API endpoint is called, taking into account the order in which this policy is applied |Optional
 |===
 
+== Example of Message Logging Policy Configured from the UI
 
-== Example of Logging Policy in XML
+To better understand how to use the Message Logging policy, consider an example "Les Vetments" company that is in the business of retail clothing and more. To run their e-commerce portal successfully, Les Vetments needs to audit every event and transaction made on their website.
+
+For example, if a customers buys or returns a product, or if the company adds or removes a product from the catalogue, the transactions must be logged. The IT manager at Les Vetments applies the Message Logging policy to their APIs to enable logging and viewing any data or transaction changes without making any modifications in the code.
+
+To log status code of the API response that is being returned to the calling client application, Les Vetments applies the Message Logging policy to the API with the following UI configurations:
```

Comment:
I've discussed it with Fede, he brings a good point that we want to avoid logging the payload. So the proposal is to log the message attributes, this is the expression #[attributes], here there is extra doc about how a messaged is composed and the attributes: https://docs.mulesoft.com/mule-runtime/4.3/intro-mule-message.

Reply:
Done! Thanks!